packages/messages/src/browser/style/notifications.css

```diff
@@ -44,7 +44,7 @@
     clear: both;
     box-sizing: border-box;
     position: relative;
-    width: 60%;
+    width: calc(100% - 10px);
```

Comment:
please revert this. it shouldn't have that width.

Reply:
added accidentally